eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -173,6 +173,7 @@ def reduceJWST(eventlabel, ecf_path='./', s2_meta=None):
                 from . import nircam as inst
             elif meta.inst == 'nirspec':
                 from . import nirspec as inst
+                log.writelog('WARNING: Are you using real JWST data? Take a look at at the flag_bg() function in nirspec.py and Issue #193 on Github!')
```

Comment:
I would instead write:
`log.writelog('WARNING: Are you using real JWST data? If so, you should edit the flag_bg() function in nirspec.py and look at Issue #193 on Github!')`